orchestration/hca_manage/common.py

```diff
@@ -23,7 +28,8 @@
 
 sam_host = {
     "dev": "https://sam.dsde-dev.broadinstitute.org",
-    "prod": "https://sam.dsde-prod.broadinstitute.org"
+    "prod": "https://sam.dsde-prod.broadinstitute.org",
+    "real_prod": "https://sam.dsde-prod.broadinstitute.org"
```

Comment:
Unrelated, this is needed to be able to manually set snapshots public in real prod via the snapshot CLI tool.